src/id3/v2/frame/mod.rs

```diff
@@ -20,12 +20,24 @@ use crate::id3::v2::items::popularimeter::Popularimeter;
 use std::convert::{TryFrom, TryInto};
 use std::hash::{Hash, Hasher};
 
-// <https://mutagen-specs.readthedocs.io/en/latest/id3/id3v2.4.0-structure.html>
-//
-// > The three byte language field, present in several frames, is used to describe
-// > the language of the frame’s content, according to ISO-639-2 [ISO-639-2].
-// > The language should be represented in lower case. If the language is not known
-// > the string “XXX” should be used.
+/// Empty content descriptor in text frame
+///
+/// Unspecific [`LanguageFrame`]s and [`EncodedTextFrame`] frames
+/// are supposed to have an empty content descriptor. Only those
+/// are currently supported as [`TagItem`]s to avoid ambiguities
+/// and to prevent inconsistencies when writing them.
+pub(super) const fn empty_content_descriptor() -> String {
+	String::new()
+}
```

Comment:
```suggestion
pub(super) const EMPTY_CONTENT_DESCRIPTOR: String = String::new();
```
Does this have to be a const fn?

Reply:
Yes, because a constant would require explicit `.clone()` operations in the code.

Reply:
You're able to swap it out with a constant in this case without clones.

Reply:
Your totally right. My assumptions were wrong.